source/core/src/main/com/csse3200/game/entities/configs/PlayerConfig.java

```diff
@@ -41,11 +41,6 @@ public class PlayerConfig extends BaseEntityConfig  {
   public String textureFilename;
   /** The texture atlas this player uses*/
   public String textureAtlasFilename;
-  
-  /** The highest possible initial health a player can have. */
-  public static final int MAX_HEALTH = 100;
-  /** The highest possible initial speed a player can have. */
-  public static final Vector2 MAX_SPEED = new Vector2(5, 5);
```

Comment:
these are the maximum initial health and speed out of *all* players, not specific to a player, was only meant to be used for the player select screen (stat bar max value). pls don't use these

Reply:
ok so how are you modifying save in that case

Reply:
@manya-k is working on that I believe

Reply:
It is done now